chat/message/src/MessageBanner/MessageBanner.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { Variant } from '@leafygreen-ui/banner';
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const baseStyles = css`
+  // Remove left wedge
+  &:before {
+    content: '';
+    background: transparent;
+  }
+
+  border-width: ${spacing[25]}px;
+  border-radius: ${spacing[600]}px;
+  border-left-color: unset;
+
+  max-width: fit-content;
+  padding: ${spacing[200]}px ${spacing[300]}px;
+`;
```

Comment:
This looks a little off with long text (the icon looks out of place) 
![Screenshot 2024-04-02 at 11 39 45 AM](https://github.com/mongodb/leafygreen-ui/assets/2414030/7a074fa7-707b-4857-a32d-a93d3edaf768)

Reply:
I missed a small part of the design here - the border radius is supposed to adjust for multiline banners.

Here's the updated version - does it look better aligned?

<img width="381" alt="Screenshot 2024-04-02 at 4 32 46 PM" src="https://github.com/mongodb/leafygreen-ui/assets/15657698/966fd211-8c9d-4c8e-996c-0903f6e26e09">

Reply:
FYI container queries ended up being a bit of a bust here so I handled this in JS instead